src/main.jsx

```diff
@@ -1,10 +1,17 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.jsx'
-import './styles/Css/index.Css'
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { Provider } from 'react-redux';
+import App from './App.jsx';
+import './styles/Css/index.Css';
```

Comment:
- [ ] On the start of your application the below error is displayed please kindly fix this so that your project runs properly on the browser :+1: It is as a result of the above highlighted line, use the below code snippet as the correct import statement for your css.
- code snippet
```javascript
import './styles/Css/index.css'
```
![rt](https://github.com/GeekyHacks/EarthquakesTracker/assets/89055502/5c34218a-d6be-455d-b902-7d355f420cb7)